ballerina/http_connection.bal

```diff
@@ -22,14 +22,11 @@ import ballerina/lang.value as val;
 # + remoteAddress - The remote address
 # + localAddress - The local address
 # + protocol - The protocol associated with the service endpoint
-public client class Caller {
-
+public isolated client class Caller {
     private ListenerConfiguration config = {};
-
-    //TODO:Make these readonly
-    public Remote remoteAddress = {};
-    public Local localAddress = {};
-    public string protocol = "";
+    private Remote remoteAddress = {};
```

Comment:
Can't we make this public and readonly? This is more like an API change

Reply:
Fixed with [4d461ab74ee46c7a163d480ff8827652e794cf51](https://github.com/ballerina-platform/module-ballerina-http/pull/765/commits/4d461ab74ee46c7a163d480ff8827652e794cf51)